knack/invocation.py

```diff
@@ -57,9 +57,9 @@ def __init__(self,
     def _filter_params(self, args):  # pylint: disable=no-self-use
         # Consider - we are using any args that start with an underscore (_) as 'private'
         # arguments and remove them from the arguments that we pass to the actual function.
-        params = dict([(key, value)
```

Comment:
Please pull this change into a separate PR.

Reply:
Done, please see https://github.com/Microsoft/knack/pull/107.